modules/overviews/spec/features/navigation_spec.rb

```diff
@@ -42,128 +42,63 @@
-      let(:project) { create(:project, enabled_module_names: %i[work_package_tracking]) }
-      let(:user) { create(:admin) }
-      let(:query) do
-        create(:query_with_view_work_packages_table,
-               project:,
-               user:,
-               name: "My important Query")
-      end
-
-      before do
-        query
-        login_as user
-      end
-
-      it "can navigate to other modules (regression #55024)" do
-        visit project_overview_path(project.id)
-
-        # Expect page to be loaded
-        within "#content" do
-          expect(page).to have_heading "Project home"
-        end
-
-        # Navigate to the WP module
-        page.find_test_selector("main-menu-toggler--work_packages").click
-
-        # Click on a saved query
-        query_menu.click_item "My important Query"
-
-        loading_indicator_saveguard
-
-        within "#content" do
-          # Expect the query content to be shown
-          expect(page).to have_field("editable-toolbar-title", with: query.name)
-
-          # Expect no page header of the Overview to be shown any more
-          expect(page).to have_no_heading "Project home"
-        end
-
-        # Navigate back to the Overview page
-        page.execute_script("window.history.back()")
-
-        # Expect page to be loaded
-        within "#content" do
-          expect(page).to have_heading "Project home"
-        end
-      end
+    within "#content" do
+      expect(page).to have_heading "Project home"
     end
   end
 
-  context "with the feature flag disabled", with_flag: { new_project_overview: false } do
-    it "can visit the overview page" do
-      visit project_path(project)
-
-      within "#menu-sidebar" do
-        click_link "Overview"
-      end
+  context "as user with permissions" do
+    let(:project) { create(:project, enabled_module_names: %i[work_package_tracking]) }
+    let(:user) { create(:admin) }
+    let(:query) do
+      create(:query_with_view_work_packages_table,
+             project:,
+             user:,
+             name: "My important Query")
+    end
 
-      within "#content" do
-        expect(page).to have_heading "Overview"
-      end
+    before do
+      query
+      login_as user
     end
 
-    context "as user with permissions" do
-      let(:project) { create(:project, enabled_module_names: %i[work_package_tracking]) }
-      let(:user) { create(:admin) }
-      let(:query) do
-        create(:query_with_view_work_packages_table,
-               project:,
-               user:,
-               name: "My important Query")
-      end
+    it "can navigate to other modules (regression #55024)" do
+      visit project_overview_path(project.id)
 
-      before do
-        query
-        login_as user
+      # Expect page to be loaded
+      within "#content" do
+        expect(page).to have_heading "Project home"
       end
 
-      it "can navigate to other modules (regression #55024)" do
```

Comment:
Same here.. I think we should keep the test, but starting from the Dashboard page.